doc/changes/changes_3.1.6.md

```diff
@@ -0,0 +1,19 @@
+# S3 Document Files Virtual Schema 3.1.6, released 2025-06-06
+
+Code name: Updated virtual-schema-common-document-files dependencies to include latest logging additions.
+
+## Summary
+
+This release updates dependencies on virtual-schema-common-document-files to include latest logging additions
+
```

Comment:
Add reference to ticket #188 and explain what are the consequences for the user (i.e. additional log messages for finding files)